### PR TITLE
feat: overlapping annotations

### LINKED ISSUE #1501: Annotations: Visually differentiate ranged annotations and point annotations when they overlap
Background
Currently, if a user were to add an annotation on a time that there exists a ranged annotation for, there is no visual distinguished that separates the two. 

Acceptance Criteria

Overlapping annotations are visually distinguishable. (This is separate from them being clickable, which will require a re-write of how the lines are stacked on z-index) 

This can be done by lowering the opacity of all our annotation click targets. Partner with @daviddkkim to get the right % of opacity and color.

### PR DESCRIPTION
**Pull request by jrenee42**
Closes #1501 

sorts the annotations so the ranges render first;
and bumps up the giraffe version to get the new changes so they render correctly

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 61 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1722)

File: src/visualization/utils/annotationUtils.ts [hunk 1]
```diff
@@ -144,6 +144,18 @@ const makeAnnotationClickHandler = (
   return clickHandler
 }
 
+// want all the ranges *before* all the points,
+// so that can click on points that overlap ranges
+const sortAnnotations = (anno1, anno2) => {
```
Comment (hoorayimhelping) [inline src/visualization/utils/annotationUtils.ts:149]: can you write a test for this? it should be pretty easy to test these cases

**Review by hoorayimhelping — COMMENTED** [review]
looks good, but how about some unit tests for the sorting function?

File: src/visualization/utils/annotationUtils.ts (hunk 1 above)
Comment (jrenee42) [reply src/visualization/utils/annotationUtils.ts]: added

**Review by hoorayimhelping — APPROVED** [review]

**Pull request merged**